otomoto/spiders/otomoto.py

```diff
@@ -16,14 +16,24 @@ def remove_spaces(x):
 
 
 def convert_to_integer(x):
-    return int(x)
-
+    y = x.split(',', 1)[0]
+    return int(y)
+
+def convert_vin(x):
+    if "?vin=" in x:
+        y = (x.split('?vin=', 1)[1]).split('#utm', 1)[0].split('&', 1)[0]
+        if len(y) != 17:
+            y = 'null'
+    else:
+        y = 'null'
+    return y
 
 class OtomotoCarLoader(ItemLoader):
     default_output_processor = TakeFirst()
 
     features_out = MapCompose(filter_out_array)
     price_out = Compose(TakeFirst(), remove_spaces, convert_to_integer)
+    vin_out = Compose(TakeFirst(), convert_vin)
```

Comment:
Maybe it's better to use `vin_in`, and then `MapCompose`?

In `price_out` I made mistake using `_out`